### ISSUE TITLE
Question about patch embedder training

### ISSUE THREAD

**Issue by kingofprank** [dc-ai-projects/DC-Gen#11]
Hi, thanks for your great jobs. I have some questions in stage 1 patch embedder training. 
1. I want to know whether the latent need to add noise like in diffusion forward process to calculate the mse loss in stage1 before sending patch embedder. 
2. If I want to train an I2V task, do I need to concatenate all the conditions in channels before feeding them into the patch embedder for training? 
3. And what is the value of mse loss after convergence in your training setting? 
 
Can you share more information? Thanks~

**Comment by CuddleSabe** [comment]
> Hi, thanks for your great jobs. I have some questions in stage 1 patch embedder training.
> 
> 1. I want to know whether the latent need to add noise like in diffusion forward process to calculate the mse loss in stage1 before sending patch embedder.
> 2. If I want to train an I2V task, do I need to concatenate all the conditions in channels before feeding them into the patch embedder for training?
> 3. And what is the value of mse loss after convergence in your training setting?
> 
> Can you share more information? Thanks~

I tried, the loss is 0.36, the middle is speedup version of OMGSR

<img width="1536" height="512" alt="Image" src="https://github.com/user-attachments/assets/2c7cb586-f745-49d9-9e9a-eb642f74110e" />

**Comment by kingofprank** [comment]
> > Hi, thanks for your great jobs. I have some questions in stage 1 patch embedder training.
> > 
> > 1. I want to know whether the latent need to add noise like in diffusion forward process to calculate the mse loss in stage1 before sending patch embedder.
> > 2. If I want to train an I2V task, do I need to concatenate all the conditions in channels before feeding them into the patch embedder for training?
> > 3. And what is the value of mse loss after convergence in your training setting?
> > 
> > Can you share more information? Thanks~
> 
> I tried, the loss is 0.36, the middle is speedup version of OMGSR
> 
> <img alt="Image" width="1536" height="512" src="https://private-user-images.githubusercontent.com/61224076/511102580-2c7cb586-f745-49d9-9e9a-eb642f74110e.png?jwt=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NjI0OTkxNTYsIm5iZiI6MTc2MjQ5ODg1NiwicGF0aCI6Ii82MTIyNDA3Ni81MTExMDI1ODAtMmM3Y2I1ODYtZjc0NS00OWQ5LTllOWEtZWI2NDJmNzQxMTBlLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNTExMDclMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjUxMTA3VDA3MDA1NlomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWZjZTliMGY3MDNlZGJkMTU5NjA3ZWUzNmIyYjFiODRkMDJiNjg3ZjgwYTlkZjgwNWRkNTFiYjdiNGQwYzI5OTImWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.7O2PFnC-ZiXtvjHB0_jZ3X_O4lzhCs_LU9jFq0jmfIk">

Thanks, the results seem good. I wonder the loss 0.36 is the mse loss for patch embedder in stage1 or diffusion mse loss in stage2. And can you share your process method about the Question 1?

**Comment by CuddleSabe** [comment]
> Thanks, the results seem good. I wonder the loss 0.36 is the mse loss for patch embedder in stage1 or diffusion mse loss in stage2. And can you share your process method about the Question 1?

0.36 is the loss of patch embedder. there is no diffusion mse loss in OMGSR because its a 1step Super-Resolution model

**Comment by yankainan** [comment]
> > Hi, thanks for your great jobs. I have some questions in stage 1 patch embedder training.
> > 
> > 1. I want to know whether the latent need to add noise like in diffusion forward process to calculate the mse loss in stage1 before sending patch embedder.
> > 2. If I want to train an I2V task, do I need to concatenate all the conditions in channels before feeding them into the patch embedder for training?
> > 3. And what is the value of mse loss after convergence in your training setting?
> > 
> > Can you share more information? Thanks~
> 
> I tried, the loss is 0.36, the middle is speedup version of OMGSR
> 
> <img alt="Image" width="1536" height="512" src="https://private-user-images.githubusercontent.com/61224076/511102580-2c7cb586-f745-49d9-9e9a-eb642f74110e.png?jwt=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NjQyMjQ4MjksIm5iZiI6MTc2NDIyNDUyOSwicGF0aCI6Ii82MTIyNDA3Ni81MTExMDI1ODAtMmM3Y2I1ODYtZjc0NS00OWQ5LTllOWEtZWI2NDJmNzQxMTBlLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNTExMjclMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjUxMTI3VDA2MjIwOVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTIxM2QyMDBkYWE5NjkwY2QwZWI3ZTcyMmQ2YWVlMGNlZWVjN2QxM2Q3ZDA5OWNhOWY3NzI0ZjZjZWE0MTQxNjAmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.iLUryHTvcw2gOFiKIonhiigYOxKOCTM-ga63RlIFOUM">

which loss you used to train output head?
